### ISSUE TITLE
Live activity showing active charging status

### ISSUE THREAD

**Issue by jverkoey** [jverkoey/ElectricSidecar#2]
This would require an iPhone app.

It's not if there's an API endpoint for getting the current charging status + estimation, but that would be necessary if we wanted to build something like this.

**Comment by leearmstrong** [comment]
From memory this also needs a server to do the polling and push notifications

**Comment by jverkoey** [comment]
Ah right. Likely more effort than it's worth then

**Comment by leearmstrong** [comment]
Ah yes, confirmed: https://developer.apple.com/documentation/activitykit/updating-and-ending-your-live-activity-with-activitykit-push-notifications

**Comment by jverkoey** [comment]
Admittedly... it would be kind of fun to see how minimal of a server could be set up to handle this... probably wouldn't need that much.

**Comment by jverkoey** [comment]
I'm already running a personal mastodon instance that I could probably just side load this onto.

**Comment by leearmstrong** [comment]
True, but would require the credentials being stored within that server?

**Comment by jverkoey** [comment]
Ah, hm. Yeah we'd definitely need to do it somehow that doesn't require that. Hmmm.
